### PR TITLE
feat: Team Section

### PR DESCRIPTION
**Pull request by Sonakshi1901**
### Your checklist for this pull request
🚨Please review the [guidelines for contributing](../CONTRIBUTING.md) to this repository.

- [X] Make sure you are requesting to **pull a topic/feature/bugfix branch** (right side). Don't request your default branch!
- [X] Make sure you are making a pull request against the **default branch** (left side). Also you should start *your branch* off *default branch*.
- [X] Check the commit's or even all commits' message styles matches our requested structure.
- [X] Check your code additions will fail neither code linting checks nor unit test.
- [X] I have added necessary documentation (if appropriate)

### Description
Added the Team section

❤️Thank you!

### Post merge checklist
- [ ] Follow steps from the [guidelines for contributing](../CONTRIBUTING.md) to this repository.
- [ ] If you are a new contributor, ping in the thread and one of the maintainers will add you to all-contributors list.

### CODE REVIEW

File: src/components/about/Executive.jsx [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+    @media (max-width: 540px){
+      display: grid;
+      grid-template-columns: auto auto;
+    }
+`;
+
+function Executive() {
+
+  return (
+    <Container>
```
Comment (riteshsp2000) [inline src/components/about/Executive.jsx:43]: Check Which container are you using. I believe Sambit has one and let's stick to that.

File: src/components/about/Team.jsx [hunk 2]
```diff
@@ -0,0 +1,216 @@
...
+            small
+          />
+        ));
+    }
+  };
+
+  return (
+    <Section>
+      <NavContainer>
+        <TeamName
```
Comment (riteshsp2000) [inline src/components/about/Team.jsx:166]: Why not loop through this as well. Reduces the code significantly

File: src/components/shared/Profile.jsx [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+    flex-direction: column;
+    justify-content: flex-start;
+    align-items: center;
+`;
+
+const Img = styled.img`
+    width: ${({ small }) => (small ? `90px` : `120px`)};
+    height: ${({ small }) => (small ? `90px` : `120px`)};
+    border-radius: 100%;
+    background: var(--color-secondary);
```
Comment (riteshsp2000) [inline src/components/shared/Profile.jsx:33]: please add the following properties so as to maintain appropriate aspect ratios
```
object-fit: cover;
object-position: center;
```

File: src/components/shared/typography/Para1.jsx [hunk 4]
```diff
@@ -8,7 +8,7 @@ export default styled.p`
     Cantarell, 'Open Sans', 'Helvetica Neue', sans-serif;
   font-style: normal;
   font-weight: normal;
-  font-size: 1.25rem;
-  line-height: 1.6;
+  font-size: 1.rem;
```
Comment (riteshsp2000) [inline src/components/shared/typography/Para1.jsx:11]: Hey don't change the font sizes, there might be inconsistencies in the design, so let's stick to the original ones itself that we created. Also keep the line height to a constant 1.6 (no unit here).

File: src/components/shared/typography/Para2.jsx [hunk 5]
```diff
@@ -9,7 +9,7 @@ export default styled.p`
   font-style: normal;
   font-weight: normal;
   font-size: 1rem;
-  line-height: 1.6;
+  line-height: 1.4;
```
Comment (riteshsp2000) [inline src/components/shared/typography/Para2.jsx:12]: Revert this change as well

**Review by riteshsp2000 — CHANGES_REQUESTED** [review]

File: src/components/about/Executive.jsx (hunk 1 above)
Comment (Sonakshi1901) [reply src/components/about/Executive.jsx:43]: ok

File: src/components/shared/Profile.jsx (hunk 3 above)
Comment (Sonakshi1901) [reply src/components/shared/Profile.jsx:33]: ok

File: src/components/shared/typography/Para1.jsx (hunk 4 above)
Comment (Sonakshi1901) [reply src/components/shared/typography/Para1.jsx:11]: ok

**Comment by riteshsp2000** [comment]
Screenshot 2021-11-14 at 2 46 18 PM

@Sonakshi1901 There is some issue with the zIndex as well. Please fix that

File: src/components/about/Team.jsx (hunk 2 above)
Comment (Sonakshi1901) [reply src/components/about/Team.jsx:166]: ok

File: src/components/about/Executive.jsx (hunk 1 above)
Comment (Sonakshi1901) [reply src/components/about/Executive.jsx:43]: I am using the one Sambit has used

File: src/components/shared/typography/Para2.jsx (hunk 5 above)
Comment (Sonakshi1901) [reply src/components/shared/typography/Para2.jsx]: ok

**Comment by Sonakshi1901** [comment]
@riteshsp2000 are there any more changes to be done?

**Comment by riteshsp2000** [comment]
@Sonakshi1901 

Actually yes. I feel that the maximum width of the container should be at max 100rem. Because in big screens its overflowing a lot and looks weird. I have a 24" monitor and its looking odd. Probably set max-wdith to 95rem

**Review by riteshsp2000 — APPROVED** [review]
